test/logger.test.js

```diff
@@ -80,24 +80,39 @@ describe('#setupNodeLogger()', () => {
       });
     });
 
-    it('should handle errors according to ecs', () => {
+    it('should handle instanceof Error according to ecs', () => {
       const log = setupNodeLogger({ json: true, level: 'info' });
 
       const errmsg = chance.string();
+      const err = new Error(errmsg);
       const stdout = capture.captureStdout(() => {
-        log.info(new Error(errmsg));
+        log.info(msg, { err });
       });
 
-      expect(JSON.parse(stdout)).to.deep.include({ message: errmsg });
+      expect(JSON.parse(stdout)).to.deep.include(
+        { error: { name: err.name, message: errmsg, stack_trace: err.stack } },
+      );
       expect(JSON.parse(stdout)).to.have.nested.property('error.stack_trace');
+
+      expect(JSON.parse(stdout)).to.deep.include({ message: msg });
```

Comment:
What's `msg`? How is it distinct from `errmsg`?

Reply:
`msg` defined in `beforeEach`. errmsg just goes into Error(..)